CRM/Custom/Form/Field.php

```diff
@@ -237,6 +237,7 @@ public function setDefaultValues() {
     else {
       $defaults['is_active'] = 1;
       $defaults['option_type'] = 1;
+      $defaults['is_search_range'] = 1;
```

Comment:
So this default is a change in behaviour outside the main PR content & I'm not 100% sure everyone would agree. 

It's more usable & less confusing but on a large database there is a real cost to creating a new searchable field.